device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -86,6 +86,11 @@
     // Keys are deviceIds. Helps with getting configs based on deviceIds
     private final Map<String, DeviceClientConfig> deviceClientConfigs = new ConcurrentHashMap<>();
 
+    // Is only set when it's not a multiplexing client.
+    private DeviceClientConfig deviceClientConfig = null;
```

Comment:
There's already a getter for the default config (the only config when not multiplexing)

```getDefaultConfig()```

Reply:
ah ! nice, will use that